src/controllers/components/cart-panel.js

```diff
@@ -35,4 +35,21 @@ export default class CartController {
     static throwError(error){
         Dispatcher.handleAction("ERROR", { error });
     }
+
+    static turnInLostEquipment(equipmentAddress){
```

Comment:
I debated where to put this. The button itself is on the view all models screen, but this isn't a model operation. It's really a different checkin workflow so I decided it would be best placed wherever the normal checkin is which is in CartController.